src/components/WorkerPoolQueueUpcomingFlowRunsList.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div class="worker-pool-queue-upcoming-flow-runs-list">
+    <FlowRunList :selected="[]" :flow-runs="filteredFlowRuns" disabled />
```

Comment:
Is it intentional to pass an unreferenced array to the selected attr?

Reply:
For whatever reason, we're doing it everywhere we use FlowRunList disabled, and `selected` is a required prop

Reply:
Hm ok, no reason to block then